File: Monstone_Android_Client/app/src/main/java/com/example/nightwingky/monstone/main_program/home/TypeAbstractViewHolder.java
package com.example.nightwingky.monstone.main_program.home;

import android.support.v7.widget.RecyclerView;
import android.view.View;

/**
 * Created by nightwingky on 17-3-18.
 */

public abstract class TypeAbstractViewHolder<T> extends RecyclerView.ViewHolder{
    public TypeAbstractViewHolder(View itemView) {
        super(itemView);
    }

    public abstract void bindHolder(T object);
}

File: kotlin_client/app/src/main/java/com/graduation/nightwingky/book_drifting/monstone/main_program/fragmentHome/TypeAbstractViewHolder.kt
package com.graduation.nightwingky.book_drifting.monstone.main_program.fragmentHome

import android.support.v7.widget.RecyclerView
import android.view.View

/**
 * Created by nightwingky on 17-3-18.
 */

abstract class TypeAbstractViewHolder<T>(itemView: View) : RecyclerView.ViewHolder(itemView) {

    abstract fun bindHolder(`object`: T)
}
